datalake_ingester/queue.py

```diff
@@ -31,7 +30,7 @@ def __init__(self, queue_name, handler=None):
 
     @classmethod
     def from_config(cls):
-        queue_name = get_config_var('queue')
+        queue_name = os.environ.get('DATALAKE_INGESTION_QUEUE')
```

Comment:
The environment variables are set by the load_config() function. They may come from the existing environment, from the default config, from a specific config, or from kwargs passed to load_config (which is how the CLI manages to get its arguments into the mix). 

Consumers of config info (e.g., this line of code) are spared from having to embrace all of that precedence business implied by this. Instead, they get that info exclusively from explicit constructor arguments or from the environment.